realis.go

```diff
@@ -1059,3 +1060,50 @@ func (r *realisClient) RollbackJobUpdate(key aurora.JobUpdateKey, message string
 	}
 	return resp, nil
 }
+
+// VariableBatchStep returns the current batch the update is in during a variable batch update.
+func (r *realisClient) VariableBatchStep(key aurora.JobUpdateKey) (int, error) {
+
+	// Query Aurora for an Update that is either paused or rolling forward
+	resp, err := r.JobUpdateDetails(aurora.JobUpdateQuery{
+		UpdateStatuses: []aurora.JobUpdateStatus{
```

Comment:
how do we handle failure path as we are filtering on ROLLING_FORWARD and ROLL_FORWARD_PAUSED

Reply:
We return an error if there's no updates matching the statuses asked for in L1082